www/include/configuration/configServers/listServers.php

```diff
@@ -190,12 +203,8 @@
     // Manage flag for changes
     $confChangedMessage = _("N/A");
     $hasChanged = false;
-    if ($config["ns_activate"] && isset($nagios_restart[$config['id']])) {
-        $hasChanged = checkChangeState(
```

Comment:
The state of this variable `$hasChanged` is always false, is it normal ?

Reply:
This variable is used to change the color (green => red) of 'Conf Changed' flag when the poller configuration has changed.

Reply:
you're right. But there is the same issue in following PR: https://github.com/centreon/centreon/pull/9259/files